ILAMB/pecan.cfg

```diff
@@ -107,11 +170,122 @@ source = DATA/Ameriflux/AMF_US-WCr.nc
 [h1: Hydrology Cycle]
 bgcolor = "#E6F9FF"
 
-[h2: Latent Heat]
+[h2: Latent Heat Flux]
 variable       = le
+<<<<<<< HEAD
+alternate_vars = LE,LE_1_1_1,LE_PI_F,LE_SSITC_TEST_1_1_1,LE_PI_F_1_1_1
```

Comment:
The `LE_SSITC_TEST_1_1_1` variable will cause us problems.

Reply:
@nocollier Oh yeah, ok. Also I decided we shouldnt be sharing the pecan.cfg file between us anyway, just the code to create it since it gets tweaked by my on my end and probably not worth tracking that.,  save us problems, except if incorrect variables show up like this!